### ISSUE TITLE
[regression] anvil-provision-server cannot find iso images

### ISSUE THREAD

**Issue by fabbione** [ClusterLabs/anvil#310]
```
[root@an-a01n01 ~]# cd /mnt/shared/
[root@an-a01n01 shared]# ls -lR
.:
total 0
drwxrwxr-x. 2 root root   6 Mar  8 05:02 archives
drwxrwxr-x. 2 root root   6 Mar  8 05:02 definitions
drwxrwxr-x. 2 root root 147 Mar  8 05:21 files
drwxrwxr-x. 2 root root   6 Mar  8 05:02 incoming
drwxrwxr-x. 2 root root   6 Mar  8 05:02 provision
drwxrwxr-x. 2 root root   6 Mar  8 05:02 temp

./archives:
total 0

./definitions:
total 0

./files:
total 9120904
-rw-r--r--. 1 root root 9339666432 Mar  8 05:13 CentOS-Stream-9-latest-x86_64-dvd1.iso
-rw-r--r--. 1 root root     372736 Mar  8 05:07 deploy1.iso
-rw-r--r--. 1 root root     372736 Mar  8 05:07 deploy2.iso
-rw-r--r--. 1 root root     372736 Mar  8 05:07 deploy3.iso
-rw-r--r--. 1 root root     372736 Mar  8 05:07 deploy4.iso
-rw-r--r--. 1 root root     372736 Mar  8 05:07 deploy5.iso

./incoming:
total 0

./provision:
total 0

./temp:
total 0
```

files are available on all nodes.

```
[root@an-a01n01 ~]# anvil-provision-server --ci-test --options
Available options
 --name          - Alphanumeric, 1~16 characters long.
 --os            - Run: 'osinfo-query os' and reference the 'Short ID' column for valid options.
 --cpu           - 1 ~ #!varaible!threads!#
 --ram           - Bytes or human readable, min is 64KiB, max is #!variable!max_ram.
 --storage-group - Name or UUID. Valid options are:
                     - Name: [Storage group 1], UUID: [685e8b92-c569-4d13-a0a8-a4c3c8ee9422], free space: [200.00 GiB]
 --storage-size  - Disk size, see above for space available
 --install-media - File name or file UUID. Available discs are:
 --driver-disc   - (optional) A driver disc to be added as a second optical drive. Valid options are above.
[root@an-a01n01 ~]# anvil-provision-server --ci-test --options
Available options
 --name          - Alphanumeric, 1~16 characters long.
 --os            - Run: 'osinfo-query os' and reference the 'Short ID' column for valid options.
 --cpu           - 1 ~ #!varaible!threads!#
 --ram           - Bytes or human readable, min is 64KiB, max is #!variable!max_ram.
 --storage-group - Name or UUID. Valid options are:
                     - Name: [Storage group 1], UUID: [685e8b92-c569-4d13-a0a8-a4c3c8ee9422], free space: [200.00 GiB]
 --storage-size  - Disk size, see above for space available
 --install-media - File name or file UUID. Available discs are:
 --driver-disc   - (optional) A driver disc to be added as a second optical drive. Valid options are above.
```

**Comment by digimer** [comment]
Reproduced.

**Comment by fabbione** [comment]
Fix verified with anvil-core-2.90-1.1923
